activerecord/lib/active_record/tasks/database_tasks.rb

```diff
@@ -247,6 +247,7 @@ def structure_load(*arguments)
 
       def load_schema(configuration, format = ActiveRecord::Base.schema_format, file = nil, environment = env, spec_name = "primary") # :nodoc:
         file ||= dump_filename(spec_name, format)
+        verbose_was, Migration.verbose = Migration.verbose, verbose?
```

Comment:
Is `Migration.verbose` thread-safe?

Reply:
Is any of this? It runs in a fork, so I think it should be fine.

Reply:
But doesn't the parallel test also have the threaded mode? I guess this don't run on those threads but before it in a fork.

Reply:
Why do we need these changes for `#load_schema`? Is this method supposed to work with migrations?
If remove these change, tests in `railties/test/application/test_test.rb` succeed. See https://github.com/rails/rails/pull/33479#issuecomment-409311852

Reply:
If we don't do this running your tests in parallel will display schema output for every single process which is noisy and unnecessary.